Documentation/articles/getting_started/1_setting_up_your_development_environment_windows.md

```diff
@@ -2,35 +2,64 @@
 
 This section provides a step-by-step guide for setting up your development environment on Windows.
 
-MonoGame can work with most .NET compatible tools, but we recommend [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) (prior versions are not supported with MonoGame 3.8.1).
+MonoGame can work with most .NET compatible tools, but we recommend [Visual Studio 2022](https://visualstudio.microsoft.com/vs/)
+
+> [!IMPORTANT]
+> Prior version of Visual Studio are not supported with MonoGame 3.8.1
```

Comment:
Only the templates are missing from older versions, its not like we provide some sort of support for it.

Reply:
The change here was to move the information already present into an alert box.  Would you suggest removing the information about prior versions completely since older versions wouldn't have the extension/templates to install anyway?

Reply:
It can also just be reworded to specify that older versions can't get the templates installed for them (and mgcb-editor launching will manually need to be added as external tool).

Reply:
Alert for prior versions removed since extension/templates are not available for prior versions so 2022 version is implied.